core/src/main/java/org/jsmart/zerocode/core/engine/mocker/RestEndPointMocker.java

```diff
@@ -25,10 +28,23 @@
 
     public static WireMockServer wireMockServer;
 
+    public static Boolean buildUrlPathEqualToPattern_FeatureFlag = true;
```

Comment:
please make the name more expressive, maybe smth like this: shouldBuildStrictUrlMatcher, or smth similar. Use only camel case.